### ISSUE TITLE
Monitored process survives gulp/gulp-nodemon exiting/restarting

### ISSUE THREAD

**Issue by csvan** [JacksonGariety/gulp-nodemon#80]
I have a very basic setup:

```
var redis = require('redis');
var app = require('express')();
var http = require('http');

// Configure the Redis client
var redisClient = redis.createClient();
redisClient.on("message", function (channel, message) {
    console.log(message);
});
redisClient.subscribe("userEvents");

// Configure the server port to use
var port = '3003';
app.set('port', port);

// Start the HTTP server
var server = http.createServer(app);
server.listen(port);
server.on('listening', function () {
    console.log('Websocket server is listening on port', port);
});

// Connect the Websocket server
var io = require('socket.io')(server);

// Configure the websocket server
io.on('connection', function (socket) {
    console.log('New user connected');
});
```

...with a very basic monitor setup:

```
var gulp = require('gulp'),
    nodemon = require('gulp-nodemon');

// Start and monitor the Frwrd realtime client server
gulp.task('default', function () {
    nodemon({
        script: 'server.js',
        ext: 'js json'
    })
});
```

There are two problem with this, however:

1. The Node process never actually restarts. This is evident because the following happens on every restart reported by Nodemon:

```
Error: listen EADDRINUSE :::3003
    at Object.exports._errnoException (util.js:814:11)
    at exports._exceptionWithHostPort (util.js:837:20)
    at Server._listen2 (net.js:1214:14)
    at listen (net.js:1250:10)
    at Server.listen (net.js:1340:5)
    at Object.<anonymous> (/home/frwrd/frwrd-realtime/server.js:18:8)
    at Module._compile (module.js:428:26)
    at Object.Module._extensions..js (module.js:446:10)
    at Module.load (module.js:353:32)
    at Function.Module._load (module.js:308:12)
```

2. The process is still running after Gulp/Gulp-Nodemon are closed. This can be confirmed through netstat:

```
tcp6       0      0 :::3003                 :::*                    LISTEN      1614/node   
```

**Comment by ericqweinstein** [comment]
I'm seeing the same problem. It can be fixed by manually setting the version of `nodemon` to 1.3.7 (when I install the latest `gulp-nodemon`, it installs 1.3.8 as a dependency). I can make a PR to fix on 1.3.7 as a temporary workaround.

**Comment by JacksonGariety** [comment]
@csvan should be fixed in nodemon 1.4.0. I'm going to publish a new version of gulp-nodemon with an updated nodemon.
